scripts/breakpad-taskcluster.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# TaskCluster build script for building Breakpad.
+# Submit a task like this to build:
+: <<'EOF'
+{
+	"created": "2016-04-09T00:49:49.410Z",
+	"deadline": "2016-04-09T01:49:49.410Z",
+	"provisionerId": "aws-provisioner-v1",
+	"workerType": "emulator-jb",
```

Comment:
It's totally arbitrary. There isn't a workerType for "run random crap", and configuring one for things like this is silly, so I just tend to use that one because it has lots of capacity.